spec/presenters/idv/by_mail/request_letter_presenter_spec.rb

```diff
@@ -67,7 +67,7 @@
     context 'when the user has a pending profile' do
       it 'returns the verify account path' do
         create(:profile, user: user, gpo_verification_pending_at: 1.day.ago)
-        expect(subject.fallback_back_path).to eq('/verify/by_mail')
+        expect(subject.fallback_back_path).to eq('/verify/by_mail/enter_code')
```

Comment:
Should this use a route helper?
```suggestion
        expect(subject.fallback_back_path).to eq(idv_verify_by_mail_enter_code_path)
```

Reply:
Yeah I think it doesn't because the helpers aren't available in the presenter, but I think I'll just include them and make the change.